apollos-church-api/config.yml

```diff
@@ -24,48 +23,26 @@ DATABASE:
         require: false
         rejectUnauthorized: false
 BUGSNAG:
-  ## Optional, but you'll need to remove references to @apollosproject/bugsnag if you don't wish to use.
   API_KEY: ${BUGSNAG_KEY}
 ROCK:
   URL: ${ROCK_URL}
   API_TOKEN: ${ROCK_TOKEN}
-  # This should match the timezone of the Rock server
-  TIMEZONE: 'America/New_York'
```

Comment:
I think this one is still required 🤔 

https://github.com/ApollosProject/apollos-apps/search?q=ROCK.TIMEZONE